Geometry/HGCalCommonData/interface/HGCalGeomParameters.h

```diff
@@ -30,39 +30,50 @@ class HGCalGeomParameters {
 public:
   HGCalGeomParameters();
   ~HGCalGeomParameters();
-  void loadGeometryHexagon(const DDFilteredView&,
-                           HGCalParameters&,
-                           const std::string&,
-                           const DDCompactView*,
-                           const std::string&,
-                           const std::string&,
-                           HGCalGeometryMode::WaferMode);
-  void loadGeometryHexagon(cms::DDFilteredView&,
-                           HGCalParameters&,
-                           const std::string&,
-                           const cms::DDCompactView*,
-                           const std::string&,
-                           const std::string&,
-                           HGCalGeometryMode::WaferMode);
-  void loadGeometryHexagon8(const DDFilteredView&, HGCalParameters&, int);
-  void loadGeometryHexagon8(cms::DDFilteredView&, HGCalParameters&, int);
-  void loadSpecParsHexagon(
-      const DDFilteredView&, HGCalParameters&, const DDCompactView*, const std::string&, const std::string&);
-  void loadSpecParsHexagon(const cms::DDFilteredView&,
-                           HGCalParameters&,
-                           const cms::DDCompactView*,
-                           const std::string&,
-                           const std::string&,
-                           const std::string&,
-                           const std::string&);
-  void loadSpecParsHexagon8(const DDFilteredView&, HGCalParameters&);
-  void loadSpecParsHexagon8(const cms::DDFilteredView&, const cms::DDVectorsMap&, HGCalParameters&, const std::string&);
-  void loadSpecParsTrapezoid(const DDFilteredView&, HGCalParameters&);
-  void loadSpecParsTrapezoid(const cms::DDFilteredView&, const cms::DDVectorsMap&, HGCalParameters&, const std::string&);
+
+  void loadGeometryHexagon(const DDFilteredView& _fv,
```

Comment:
Hello @bsunanda, I see you are loading `DDFilteredView/CompactView` both from `Core` and `DDCMS`. Do you have a list of missing methods (or compiler errors) that you get by using `DDCMS` instances only?

Reply:
@vargasa The idea is that I would like to load geometry for either DDD or DD4Hep scenario. The difference is created by a run time flag which is under control of an ERA. So when this flag is set it loads geometry using DDCMS call, otherwise using call to Core. The 2 scenarios DDD vs DD4Hep need to coexist for a while. I do not want to have 2 libraries for core and DDCMS functionalities at this moment.